packages/vm/examples/l2-playground/fetch-polygon.ts

```diff
@@ -0,0 +1,127 @@
+const fetch = require('node-fetch')
+import VM from '../../src'
+import Common, { CustomChain, Hardfork } from '@ethereumjs/common'
+import { Transaction } from '@ethereumjs/tx'
+import { Account, Address, toBuffer } from 'ethereumjs-util'
+import { BN } from 'ethereumjs-util/dist/externals'
+import { Block } from '@ethereumjs/block'
+import { mumbaiData } from './polygon-mumbai-data'
+
+const url = 'https://polygon-mumbai.g.alchemy.com/v2/EomkJjyUZ_sLzKWuaExlMwcFQZ-hDpjZ'
```

Comment:
Not sure if this is only meant to be temporary anyhow, just that we start some early exchange here: we for sure can't expose some private Alchemy API key here which is then used in ways (quantities) we can't control (maybe you should generally not expose in a PR, might have unwanted side effects already).

I wonder if we want to change the `ethers/abi` dev dependency we have already in the VM to a generic `ethers` dependency. I guess this will likely be useful for future examples as well and then people can use their own providers and/or API keys, eventually with the need to adopt an example a bit.

WDYT?

Reply:
Si, thanks for the catch, indeed it is temporary and I did not mean to push this to GH

> I wonder if we want to change the ethers/abi dev dependency we have already in the VM to a generic ethers dependency. I guess this will likely be useful for future examples as well and then people can use their own providers and/or API keys, eventually with the need to adopt an example a bit.

Yah I like this, let's do it - Also, for the time being, I will add these variables (like API key from alchemy or an account private key) through environment variables